packages/blocks/src/embed-loom-block/embed-loom-block.ts

```diff
@@ -220,7 +219,7 @@ export class EmbedLoomBlockComponent extends EmbedBlockElement<
             </div>
           </div>
 
-          <embed-card-caption .block=${this}></embed-card-caption>
+          <block-caption-editor .block=${this}></block-caption-editor>
```

Comment:
A lot of blocks has this `block-caption-editor` and `affine-block-selection` and some related logics. Can we do better?

Reply:
Absolutely! I believe we can handle this in a standalone PR.